apps/avatax/package.json

```diff
@@ -87,6 +87,7 @@
     "eslint": "../../node_modules/eslint",
     "eslint-config-saleor": "workspace:*",
     "eslint-plugin-neverthrow": "^1.1.4",
+    "eslint-plugin-simple-import-sort": "12.1.1",
```

Comment:
why dont you add this to eslint-config-saleor package for every app?

Reply:
Done - I also explicitly added `eslint-plugin-import` to pkg json as previously we have rule enabled from this package. I also added recommended rules from example https://github.com/lydell/eslint-plugin-simple-import-sort#example-configuration